R/test-files.R

```diff
@@ -27,16 +27,16 @@ test_env <- function() {
 #' @return the results as a "testthat_results" (list)
 #' @export
 test_dir <- function(path, filter = NULL, reporter = "summary",
-                     env = test_env(), ..., load_helpers = TRUE) {
+                     env = test_env(), ..., encoding = "unknown", load_helpers = TRUE) {
   if (load_helpers) {
     source_test_helpers(path, env)
   }
   paths <- find_test_scripts(path, filter, ...)
 
-  test_files(paths, reporter = reporter, env = env, ...)
+  test_files(paths, reporter = reporter, env = env, encoding = encoding, ...)
 }
 
-test_files <- function(paths, reporter = "summary",
+test_files <- function(paths, reporter = "summary", encoding = "unknown",
```

Comment:
The dots `...` currently are not used by `test_files()`. They weren't before, but as we're touching the code now, it's better to clean this up.